src/utils/server-utils.ts

```diff
@@ -270,3 +278,119 @@ export async function getLayersAvailableDates(): Promise<AvailableDates> {
 
   return merge({}, ...layerDates);
 }
+
+/**
+ * Format value from featureInfo response based on LabelType provided
+ *
+ * @return a formatted string
+ */
+function formatFeatureInfo(value: string, type: LabelType): string {
+  if (type === LabelType.Date) {
+    return `${moment(value).utc().format('MMMM Do YYYY, h:mm:ss')} UTC`;
+  }
+
+  return value;
+}
+
+/**
+ * Executes a getFeatureInfo request
+ *
+ * @return object of key: string - value: string with formatted values given label type.
+ */
+async function runFeatureInfoRequest(
+  url: string,
+  wmsParams: requestFeatureInfo,
+  layers: WMSLayerProps[],
+): Promise<{ [name: string]: string }> {
```

Comment:
On line 318 the API result is typed well, so we should be able to provide a good type as the output for this function if we properly type the `reduce` functions used here. What do you think? We should always try to make our types as strong as possible.